src/preferences.html

```diff
@@ -46,68 +47,78 @@
+                <p><i class="far fa-bell"></i> <span data-i18n="$Preferences.notification">Notification</span></p>
                 <label class="switch"><input type="checkbox" id="notification" name="notification"><span class="slider round"></span></label>
             </div>
             <div class="flex-box">
-                <p data-i18n="$Preferences.allowRecurringNotifications">Allow recurring notifications</p>
+                <p><i class="fas fa-redo"></i> <span data-i18n="$Preferences.allowRecurringNotifications">Allow recurring notifications</span></p>
                 <label class="switch"><input type="checkbox" id="repetition" name="repetition"><span class="slider round"></span></label>
             </div>
             <div class="flex-box mb-2">
-                <p data-i18n="$Preferences.minutesBetweenNotifications">Minutes between notifications</p>
+                <p><i class="fas fa-stopwatch"></i><span data-i18n="$Preferences.minutesBetweenNotifications">Minutes between notifications</span></p>
                 <input type="number" name="notifications-interval" id="notifications-interval" min="1" max="30" value="5" required onblur="this.value = this.checkValidity() ? this.value : 5">
             </div>
+            
+        </section>
+
+        <section>
+            <div class="section-title" data-i18n="$Preferences.appBehavior">App Behavior</div>
             <div class="flex-box">
-                <p data-i18n="$Preferences.startOnLogin">Start on Login</p>
-                <label id='start-at-login' class="switch"><input type="checkbox" name="start-at-login"><span class="slider round"></span></label>
+                <p><i class="fas fa-globe"></i><span data-i18n="$Preferences.language">Language</span></p>
+                <select class="selects-right" id="language" name="language">
+                </select>
             </div>
             <div class="flex-box">
-                <p data-i18n="$Preferences.countTodayOnTotals">Count Today in Totals</p>
-                <label id='count-today' class="switch"><input type="checkbox" name="count-today"><span class="slider round"></span></label>
-            </div>
-            <div class="flex-box mb-2">
-                <p data-i18n="$Preferences.overallBalanceStart">Overall Balance Start Date</p>
-                <label id='overall-balance-start-date'><input type="date" class="date-right" name="overall-balance-start-date"></label>
+                <p><i class="far fa-play-circle"></i><span data-i18n="$Preferences.startOnLogin">Start on Login</span></p>
+                <label id='start-at-login' class="switch"><input type="checkbox" name="start-at-login"><span class="slider round"></span></label>
             </div>
             <div class="flex-box">
-                <p data-i18n="$Preferences.minimizeToTray">Minimize button should minimize to tray</p>
+                <p></i><span data-i18n="$Preferences.minimizeToTray">Minimize button should minimize to tray</span></p>
```

Comment:
I think it's a little bit strange for the minimize/close to not have an icon. How about:

```suggestion
                <p><i class="far fa-window-maximize"><span data-i18n="$Preferences.minimizeToTray">Minimize button should minimize to tray</span></p>
```

![image](https://user-images.githubusercontent.com/846063/97367978-ce424300-1888-11eb-829d-41eec7649352.png)

I know it says maximize, but I didn't like the minimize version hahahah

Reply:
Maybe it's a bit weird, but I think these ones is so long, that they can't be with icons

Reply:
Hmm... That's a fair point. We can always increase the width of the window anyway (it's already a problem for some languages that have some bigger strings... But I think for now, It's ok to leave without the icon.